presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -328,12 +328,21 @@ public boolean isFinished()
     }
 
     @Override
+    public ListenableFuture<?> isBlocked()
```

Comment:
why is that needed?
It is handled by `Driver.isBlocked(Operator)`

I opt for removing this method from inteface.